### PR TITLE
Update ome-version-comparison.md

### PR DESCRIPTION
**Pull request by samschan-ms**
Added KB article that describes how to determine and migrate AD RMS to Azure RMS.
@KCCross please review.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/ome-version-comparison.md [hunk 1]
```diff
@@ -32,19 +32,9 @@ This article is part of a larger series of articles about Office 365 Message Enc
 
 Exchange Online includes Information Rights Management (IRM) functionality that provides online and offline protection of email messages and attachments. By default, Exchange Online uses Azure Azure Information Protection. However, your organization may have configured Exchange Online IRM to use on-premises Active Directory Rights Management Service (AD RMS). AD RMS support in Exchange Online is retiring. Instead, Azure Information Protection will replace AD RMS entirely.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-version-comparison.md]:
```suggestion
Exchange Online includes Information Rights Management (IRM) functionality that provides online and offline protection of email messages and attachments. By default, Exchange Online uses Azure Information Protection. However, your organization may have configured Exchange Online IRM to use on-premises Active Directory Rights Management Service (AD RMS). AD RMS support in Exchange Online is retiring. Instead, Azure Information Protection will replace AD RMS entirely.
```

File: microsoft-365/compliance/ome-version-comparison.md [hunk 2]
```diff
@@ -32,19 +32,9 @@ This article is part of a larger series of articles about Office 365 Message Enc
...
-If you've already set up Azure Information Protection but you're not using it, enable the service using Exchange Online PowerShell. On your local computer, using a work or school account that has global administrator permissions in your organization, [connect to Exchange Online PowerShell](https://docs.microsoft.com/powershell/exchange/connect-to-exchange-online-powershell) in a Windows PowerShell window.
-
-To enable Azure Information Protection, use the Set-IrmConfiguration cmdlet by typing the following command.
-
-```powershell
-Set-IrmConfiguration -AzureRMSLicensingEnabled $true
-```
-
-If your organization has not yet set up Azure Information Protection, you'll need to migrate from AD RMS to Azure Information Protection. For instructions, see [Migrating from AD RMS to Azure Information Protection](https://docs.microsoft.com/azure/information-protection/migrate-from-ad-rms-to-azure-rms).
+See [KB article 5001237]https://support.microsoft.com/help/5001237 to assess whether this impacts your organization.  The article provides recommendation on migation options.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-version-comparison.md]:
```suggestion
See the topic [How to migrate AD RMS to Azure RMS in Exchange Online](https://support.microsoft.com/help/5001237) to assess whether this deprecation impacts your organization. The article provides recommendations on migration options.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@samschan-ms

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by JohanFreelancer9** [comment]
@samschan-ms - please commit my second suggestion as well.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**